pkg/volume/iscsi/iscsi_util.go

```diff
@@ -86,10 +88,34 @@ func updateISCSIDiscoverydb(b iscsiDiskMounter, tp string) error {
 			}
 		}
 	}
+
 	return nil
 }
 
+//updateISCSINodeTimeouts will update the timeout settings with the
+//values provided in Persistent Volume object
+func updateISCSINodeTimeouts(b iscsiDiskMounter, tp string) {
+	//Updating the iSCSI timeouts settings to keep volume in RW state in the node
+	//down scenario i.e when the node goes down K8s take 5min to schedule the node
+	//until that time we can keep volume in RW state by updating these timeouts
+	//setting value to 300+ seconds. This updation will execute if PV object
+	//contains a valid key, value pair
+	if b.iscsiTimeouts != nil {
+		for _, iscsiSetting := range b.iscsiTimeouts {
+			if iscsiSetting.Name != "" && iscsiSetting.Value != "" {
+				out, err := b.exec.Run("iscsiadm", "-m", "node", "-p", tp, "-T", b.Iqn, "-I", b.Iface, "-o", "update", "-n", string(iscsiSetting.Name), "-v", iscsiSetting.Value)
```

Comment:
This just blindly sets **any** parameters from the annotation to `iscsiadm`. It does not check if it's timeout related. This should be validated somewhere and you should allow only very specific keys.